### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ Kubernetes documentation is organized into several categories.
     - for people who want to contribute code to kubernetes
     - covers development conventions
     - explains current architecture and project plans
+  - ** Service Documentation***
```
Comment (a-robinson) [inline README.md]: One too many asterisks on the end :) It's not displaying properly

**Comment by filbranden** [comment]
This branch was created in the upstream GoogleCloudPlatform/kubernetes. The same happened with Clayton earlier related to using GitHub on a phone interface. These branches are annoying since everyone that does `git fetch` ends up getting a copy of them. Can I ask you to be careful not to create new work branches in the main project? Thanks!

File: README.md (hunk 1 above)
Comment (brendandburns) [inline README.md]: done.

**Comment by brendandburns** [comment]
@filbranden yeah, sorry, that's due to the github "edit then PR" interface, I'll make sure I do online editing in my own project, not he main project in the future.

Comment addressed, ptal.

--brendan

**Comment by a-robinson** [comment]
LGTM, thanks

**Pull request merged**